app/controllers/ticket_controller.rb

```diff
@@ -2,25 +2,28 @@ class TicketController < ApplicationController
   before_action :set_ticket, only: []
 
   def index     
-
     @tickets = Ticket.where(ticket_desk_id: params[:ticket_desk_id]).where(cinema_hall_id: params[:cinema_hall_id])    
     render json: @tickets
   end
 
   def show
     render json:[]
   end
+  
+  def new
+    @ticket = Ticket.new
+  end
 
   def create
     #Cant buy if there is not such ticketdesk or cinemahall
     #Calling multiple methods using .send and []
       if ticket_available?
         if Ticket.exists?(:paid => false)  
-          Ticket.where(:paid => false).update_all(paid: true)
-          render :json => "You've paid for ticket that You had booked."
+          confirm_reservation()
```

Comment:
When there is no arguments, you can execute method without bracket - `confirm_reservation`